FastCaloSimAnalyzer/FastCaloGpu/src/CaloGpuGeneral.cxx

```diff
@@ -70,6 +71,8 @@ void* CaloGpuGeneral::Rand4Hits_init( long long maxhits, unsigned short maxbin,
 void CaloGpuGeneral::Rand4Hits_finish( void* rd4h ) {
   #ifdef USE_STDPAR
   CaloGpuGeneral_stdpar::Rand4Hits_finish( rd4h );
+  #elif defined (USE_ALPAKA)
```

Comment:
stupid github won't let me add a comment on a line you haven't touched... so on line 63, need to add another #elseif block for USE_ALPAKA

Reply:
Fixed